backend/offchain/types/__init__.py

```diff
@@ -257,7 +271,7 @@ def new_payment_request(
     cid: typing.Optional[str] = None,
 ) -> CommandRequestObject:
     return CommandRequestObject(
-        cid=cid or str(uuid.uuid4()),
+        cid=cid or generate_cid(),
```

Comment:
I don't see how this resolve the issue you mention - you extracted `str(uuid.uuid4())` into method in separate file... what do I missing?

Reply:
In all the other cases, where we sent the reference ID instead of generating a new cid, my change fixes things.
In this line specifically, I don't fix but make it better (I believe)  - it makes sense to me that we will have only one place that knows how to generate a new cid.